config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
-  resources :posts
-  get 'static_pages/home'
-  get 'static_pages/help'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'users/show'
```

Comment:
remove because not use

Reply:
![Screenshot from 2021-09-10 12-56-09](https://user-images.githubusercontent.com/87750131/132801769-5e6b11b1-2ee6-420f-9cd7-2058fb971bd1.png)